### ISSUE TITLE
Fix Control Tab Order

### ISSUE THREAD

**Issue by adanvdo** [adanvdo/YT-RED-UI#77]
Properly sort the tab order of all usable controls.

**Comment by adanvdo** [comment]
I did my best to set the Tab order of controls, but since the Input Panel and Control Panels are UserControls, you cannot tab through them from the mainform. Tabbing through controls in those panels now happens in the proper order.
